apps/subscriber-app/src/app/notificationBanner.tsx

```diff
@@ -38,4 +54,14 @@ export const NotificationStyles = styled.div`
     margin: 0 !important;
     padding: 0 !important;
   }
+
+  display: flex;
+
+  .verification-code-button {
+    margin-left: 8px;
+    margin-top: -4px;
```

Comment:
Hi @jonathanweyermann, is there any reason we try to avoid using the styled-component here? I remember the GoA CSS variables have been input to the app. I would suggest double checking it.

Reply:
I don't think I can get 4px using CSS variables, but If I can I'd like to fix it in a follow up PR - this pr contains from preliminary styling that I probably have to fix later anyway